src/index.js

```diff
@@ -51,3 +56,7 @@ window.onSceneLoad = function() {
 
   scene.components["networked-scene"].connect();
 };
+
+window.onConnect = function() {
+  document.getElementById("loader").style.display = "none";
+};
```

Comment:
Maybe create a global `App` namespace instead of continuing to pollute `window`?
```
window.App = {
    onSceneLoad: ...
    onConnect: ...
};
```